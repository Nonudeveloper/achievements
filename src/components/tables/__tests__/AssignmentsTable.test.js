import React from "react";
import { createShallow } from "@material-ui/core/test-utils";
import AssignmentsTable from "../AssignmentsTable";
import sinon from "sinon";

describe("<AssignmentsTable>", () => {
  let shallow;
  let dispatch;

  beforeEach(() => {
    shallow = createShallow();
    dispatch = sinon.spy();
  });

  it("should check snapshot", () => {
    const wrapper = shallow(
      <AssignmentsTable dispatch={dispatch} isInstructor={false} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
