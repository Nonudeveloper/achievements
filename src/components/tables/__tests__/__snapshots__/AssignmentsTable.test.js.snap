// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AssignmentsTable> should check snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(AssignmentsTable)
    dispatch={[Function]}
    isInstructor={false}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "narrowCell": "AssignmentsTable-narrowCell-1",
      },
      "dispatch": [Function],
      "isInstructor": false,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "narrowCell": "AssignmentsTable-narrowCell-1",
        },
        "dispatch": [Function],
        "isInstructor": false,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "context": Object {},
  },
}
`;
